linera-views/src/scylla_db.rs

```diff
@@ -29,14 +29,14 @@ use thiserror::Error;
 type ScyllaDbClientPair = (Session, Vec<u8>);
 
 /// We limit the number of connections that can be done.
-/// TODO: Put it as parameter when PR 931.
-const MAX_CONNECTIONS: usize = 10;
+const SCYLLA_DB_MAX_CONCURRENT_QUERIES: usize = 10;
```

Comment:
Here and in RocksDb, it looks like we're using this constant for two different things:
* The default value of `preferred_num_concurrent_requests` in production.
* The number of max concurrent queries for testing.

Can we use two different constants? Perhaps the test one(s) can be tagged `#[cfg(test)]`.